### ISSUE TITLE
Expand the set of example runcards to showcase more diverse applications

### ISSUE THREAD

**Issue by jmarshrossney** [wilsonmr/anvil#78]
E.g. 
- [ ] Checking bare mass is reproduced when training against non-interacting theory
- [ ] Studying intermediate layers as probability distributions
- [ ] Looking at the weights and biases of neural networks
- [ ] Measuring the integrated autocorrelation time by forcing `sample_interval=1`, versus allowing it to be set automatically to speed up generation of a representative sample
- [ ] Showcase different flow models
- [ ] Compare batch normalisation with global rescaling (fixed / learnable scale parameter)
